.github/actions/spelling/expect.txt

```diff
@@ -0,0 +1,493 @@
+Abf
+Abh
+acbf
+addressparser
+AHyperlink
+alekstorm
+algo
+allcaps
+amd
+amlog
+anonymize
+Aopen
+api
+applypatch
+apps
+AQws
+args
+argv
+ARIo
+Arnfj
+asciidoc
+attrs
+autocrlf
+autolearn
+automerge
+autosquash
+avarab
+aws
+backend
+basedon
+BAYES
+bbox
+BCAE
+bcbb
+bfbde
+BFk
+bigfilethreshold
+bigint
+Bjarmason
+BKBc
+BKBd
+BKL
+blockquote
+blocksource
+BNew
+bordercolor
+borderopacity
+brk
+BTr
+buf
+bugfix
+busybox
+BXx
+CABPp
+cacheinfo
+callbackfn
+callchain
+CBGFIg
+ccd
+CDcd
+CEu
+charset
+chdir
+CIimap
+CIPh
+CIsmtp
+CIsmtphost
+CIsmtpopts
+CIsmtppass
+cleanbranch
+cmds
+cmp
+cmt
+coccinelle
+codeaurora
+codepath
+colabti
+commonjs
+compat
+concat
+config
+Cpjh
+creativecommons
+cscscscc
+Ctw
+cummings
+CURLOPT
+cwd
+cygwin
+CYKD
+daf
+dasharray
+dbd
+dcmitype
+dco
+dddf
+deduplicate
+destructuring
+devnull
+devops
+DFEEDBEEF
```

Comment:
It's possible to add more patterns to reduce the amount of things like this. I stopped when it got to a manageable level.